homeassistant/components/dexcom/sensor.py

```diff
@@ -41,6 +42,10 @@ def __init__(
         """Initialize the sensor."""
         super().__init__(coordinator)
         self._attr_unique_id = f"{username}-{key}"
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, username)},
```

Comment:
I know this was a review comment, but... can the user name change in any way?

If so, this cannot be used...

../Frenck

Reply:
According to the code owner it's the only unique thing available. Other solution would be entry id

Reply:
In case of a service like this, I would consider reverting the last change and stick with the entry_id.